config.json

```diff
@@ -612,6 +612,18 @@
       "unlocked_by": "rotational-cipher",
       "uuid": "d36ce010-210f-4e9a-9d6c-cb933e0a59af"
     },
+    {
+      "core": false,
+      "difficulty": 6,
+      "slug": "alphametics",
+      "topics": [
+        "mathematics",
+        "logic",
+        "conditionals"
+      ],
+      "unlocked_by": null,
```

Comment:
If you'd like to suggest an exercise this could be unlocked by, that would be great :) It has to be a core exercise and we normally choose a core exercise which has similar topics. Maybe `secret-handshake` could be a good suggestion for this one? It's sort of to do with logic and unlocks some other exercises to do with logic: `robot-simulator`, `wordy`, and `forth`. If you don't want to fill this in now then that's fine, we can open a separate issue for it :)

In case you're not familiar with the exercism structure, users will first only have a few easy exercises plus the first core exercise available to them. Once they complete that core exercise the next core exercise will become available to them plus any exercises that the first core exercise unlocks :)

Reply:
Sorry @sjwarner-bp, I didn't see that you'd already put a comment about this! :P

Reply:
Don't worry about it, your feedback is a bit more detailed 😉